mrp_packaging_default/models/stock_move.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2023 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+
+
+from odoo import api, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    @api.model
+    def _packaging_vals_from_bom_line(self, vals):
+        """Fill vals with packaging info from BoM line."""
+        try:
+            bom_line = self.env["mrp.bom.line"].browse(vals["bom_line_id"])
+        except KeyError:
+            # No BoM line, nothing to do
+            return
+        vals.update(
+            {
+                "product_packaging_id": bom_line.product_packaging_id.id,
+                "product_packaging_qty": bom_line.product_packaging_qty,
+            }
+        )
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        """Inherit packaging from BoM line."""
+        for vals in vals_list:
+            self._packaging_vals_from_bom_line(vals)
+        return super().create(vals_list)
+
+    def write(self, vals):
+        """Inherit packaging from BoM line."""
+        self._packaging_vals_from_bom_line(vals)
+        return super().write(vals)
```

Comment:
On this I'm wondering, do we necessarily want to force the packaging on all the moves related to a BOM if module is installed? Shouldn't we have an option on the BOM to activate/deactivate the behaviour?

Reply:
If you don't want packaging info to be transferred from BoM lines to stock moves, then you should either:
- Don't enable packagings in your DB.
- Don't install this module.
- Don't add packaging info in the BoM line.

At first sight, I don't see a real world use case for that option, so for starting, I prefer to have a simpler MVP implementation.

Reply:
As your module is Alpha, that's fine as long as it's not messing with CI (which seems to be the case).